notebooks/customizing_potentials_cookbook.ipynb

```diff
@@ -158,6 +157,33 @@
       "execution_count": 0,
       "outputs": []
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "lHhvM9_nZ9TT",
+        "colab_type": "text"
+      },
+      "source": [
+        "#Prerequisites\n",
```

Comment:
Might be worth having a title + author in a cell above this?

Reply:
Done. Is there anything in particular I should say other than "This cookbook was contributed by Carl Goodrich."??